src/config/theme/components/cases.ts

```diff
@@ -3,10 +3,14 @@ const cases = {
     container: {
       display: 'grid',
       gridGap: 5,
-      gridTemplateColumns: ['repeat(2, 1fr)', 'repeat(6, 1fr)'],
+      gridTemplateColumns: [
+        'repeat(2, minmax(0, 1fr))',
+        'repeat(6, minmax(0, 1fr))',
+      ],
     },
     mainStat: {
       gridColumn: ['span 1', 'span 3'],
+      padding: '24px 12px',
```

Comment:
hey! remember that each unit of size = 8px so can you change this to be `py: 3` and `py: 2` please? I want all sizes to be multiples of 8.

Reply:
Good catch!
Alright, re-review